pkg/agent/flowexporter/ipfix/ipfix_registry.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ipfix
+
+import (
+	ipfixentities "github.com/vmware/go-ipfix/pkg/entities"
+	ipfixregistry "github.com/vmware/go-ipfix/pkg/registry"
+)
+
+var _ IPFIXRegistry = new(ipfixRegistry)
+
+// IPFIXRegistry interface is added to facilitate unit testing without involving the code from go-ipfix library.
+type IPFIXRegistry interface {
+	LoadRegistry()
+	GetInfoElement(name string, enterpriseID uint32) (*ipfixentities.InfoElement, error)
+}
```

Comment:
I am fine with this change, but was there a strong incentive to introduce that interface for unit testing? I would think that operations on the go-ipfix registry are not really complex / costly / platform-dependent / etc.

Reply:
This was suggested by @srikartati that for unit tests we want to test Antrea code only. We will add integration tests with go-ipfixlib later.

Reply:
Yes, the main thought was keeping the behavior consistent across all ipfix packages used. I felt that maybe the operations in ipfix registry are simple enough to call them in to directly in unit tests but we create ~1000 objects for information elements as part of loading registries that correspond to ~50KB of memory and the registry could evolve in the future. 

Currently, integration with ipfix library is tested through the e2e test. We want to enhance the coverage through integration tests.

Reply:
It's fine by me. We can see how this needs to evolve in the future.